src/poliastro/twobody/orbit.py

```diff
@@ -610,6 +610,71 @@ def parabolic(
         ss = ClassicalState(attractor, p, 1.0 * u.one, inc, raan, argp, nu)
         return cls(ss, epoch, plane)
 
+    @classmethod
+    @u.quantity_input(alt=u.m, inc=u.rad, argp=u.rad, raan=u.rad, arglat=u.rad)
+    def frozen(
+        cls,
+        alt,
+        inc=Earth.critical_inclinations[0],
+        argp=Earth.critical_argps[0],
+        raan=0 * u.deg,
+        arglat=0 * u.deg,
+        epoch=J2000,
+        plane=Planes.EARTH_EQUATOR,
+    ):
+        """Return frozen 'Orbit'.
+
+        Parameters
+        ----------
+        alt : ~astropy.units.Quantity
+            Altitude over surface.
+        inc : ~astropy.units.Quantity, optional
+            Inclination, default to critical value.
+        argp : ~astropy.units.Quantity, optional
+            Argument of the pericenter, default to critical value.
+        raan : ~astropy.units.Quantity, optional
+            Right ascension of the ascending node, default to 0 deg.
+        arglat : ~astropy.units.Quantity, optional
+            Argument of latitude, default to 0 deg.
+        epoch: ~astropy.time.Time, optional
+            Epoch, default to J2000.
+        plane : ~poliastro.frames.Planes
+            Fundamental plane of the frame.
+        """
+        if alt <= 0:
+            raise ValueError(
+                "The semimajor axis may not be smaller that the earth's radius"
+            )
+        a = Earth.R + alt
+
+        critical_argp = cls._find_closest_value(argp, Earth.critical_argps)
+        if np.isclose(argp, critical_argp, 1e-8, 1e-5 * u.rad):
+            return cls._frozen_critical_argp(a, inc, argp, raan, arglat, epoch, plane)
+
+        critical_inclination = cls._find_closest_value(inc, Earth.critical_inclinations)
+        if np.isclose(inc, critical_inclination, 1e-8, 1e-5 * u.rad):
+            return cls._frozen_critical_inclination(
+                a, inc, argp, raan, arglat, epoch, plane
+            )
+
+        raise ValueError("Can't create a frozen orbit from given arguments")
+
+    @classmethod
+    def _find_closest_value(cls, value, values):
+        critical_values = [critical_arg.value for critical_arg in values]
+        index = np.abs(np.asarray(critical_values) * u.rad - value).argmin()
+        return values[index]
+
+    @classmethod
+    def _frozen_critical_argp(cls, a, inc, argp, raan, arglat, epoch, plane):
+        ecc = -Earth.J3 * Earth.R * np.sin(inc) / 2 / Earth.J2 / a
+        return cls.from_classical(Earth, a, ecc, inc, raan, argp, arglat, epoch, plane)
+
+    @classmethod
+    def _frozen_critical_inclination(cls, a, inc, argp, raan, arglat, epoch, plane):
+        ecc = 5.5e-2 * u.one  # Same as the moon
```

Comment:
Let's have the formula to calculate this eccentricity in `poliastro.util` as well.